package.json

```diff
@@ -9,6 +9,8 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@fortawesome/free-solid-svg-icons": "^6.4.2",
+    "@fortawesome/react-fontawesome": "^0.2.0",
```

Comment:
Unused dependencies or am I missing something?

Reply:
Yeah that is unused I will remove since ended up using shadcn